app/src/main/java/mozilla/lockbox/store/ClipboardStore.kt

```diff
@@ -48,5 +51,16 @@ open class ClipboardStore(
     private fun addToClipboard(label: String, str: String) {
         val clip = ClipData.newPlainText(label, str)
         clipboardManager.primaryClip = clip
+        clearClipboard(clipDataClearDelay, str)
+    }
+
+    private fun clearClipboard(delay: Long, clearStr: String) {
+        Handler().postDelayed({
```

Comment:
How does a `Handler` behave when the app is backgrounded or killed? The answer to this may have privacy/security implications that @linuxwolf may be interested in.

Reply:
if app is in background handler is working, if app is killed everything is killed including that handler

Reply:
There are limits to how much we can/should keep ourselves alive.  Absent a more system-managed mechanism (such as iOS has), this seems the best compromise between respecting user information and respecting the user's device resources (battery/CPU/memory).

Reply:
Exactly the responses I was hoping for. Thanks